aggregators/aggregator.go

```diff
@@ -555,11 +555,14 @@ func (a *Aggregator) harvestForInterval(
 			continue
 		}
 		if eventsTotal < 0 {
-			a.cfg.Logger.Warn(
-				"unexpectedly failed to harvest all collected events",
+			fields := append([]zap.Field{
 				zap.Duration("aggregation_interval_ns", ivl),
 				zap.ByteString("id", cmID[:]),
 				zap.Float64("remaining_events", eventsTotal),
```

Comment:
```suggestion
				zap.Duration("aggregation_interval_ns", ivl),
				zap.Float64("remaining_events", eventsTotal),
```
I think `cmID` as byte string will mostly be noise and we should remove it.